### PR TITLE
Update bootstrap classes from V3 to v4 - Fixes IE11 issues

### PR DESCRIPTION
**Pull request by ShmoogleOsukami**
Updating the bootstrap classes from their V3 versions to V4 fixes the issue with IE11.

I've also added better fix for the 100% height issue on low content pages as suggested by cycy.

### CODE REVIEW

**Comment by barton2526** [comment]
Will this conflict with #176 @ShmoogleOsukami ?

**Comment by RoboticMind** [comment]
Did a test on IE11 to confirm that it works. Does seem to work 
image

**Comment by ShmoogleOsukami** [comment]
> Will this conflict with #176 @ShmoogleOsukami ?

Yes I believe this will conflict with that PR.

File: _includes/_footer.htm [hunk 1]
```diff
@@ -1,9 +1,9 @@
-<footer class="gridcoin-footer h-100" data-background-color="black">
```
Comment (RoboticMind) [inline _includes/_footer.htm]: why was the h-100 class removed here? Isn't it a valid bootstrap 4 class? See https://getbootstrap.com/docs/4.0/utilities/sizing/

File: _layouts/layout.htm [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-	{{ content }}
-	{% include _footer.htm %}
+    {% include _head.htm %}
+    <body class="h-100">
+    {% include _header.htm %}
+    <div class="min-h-80">{{ content }}</div>
+    {% include _footer.htm %}
 
 	<!--   Core JS Files   -->
 	<script src="/assets/js/core/jquery-3.4.1.min.js" type="text/javascript"></script>
```
Comment (RoboticMind) [inline _layouts/layout.htm]: Indentation changes make the Core JS Files Part not line up with the rest of the HTML

File: _includes/_footer.htm (hunk 1 above)
Comment (ShmoogleOsukami) [reply _includes/_footer.htm]: The h-100 was part of the previous low content fix, It's not required anymore.

File: _layouts/layout.htm (hunk 2 above)
Comment (ShmoogleOsukami) [reply _layouts/layout.htm]: Pushed a fix for the indentation.

Comment (RoboticMind) [reply _layouts/layout.htm]: looks good

**Pull request merged**